test/src/test/resources/fixtures/hydration/hydration-input-is-empty-list.yml

```diff
@@ -20,13 +20,19 @@ overallSchema:
       }
       type Issue {
         id: ID
-        authors: [User] => hydrated from UserService.usersByIds(ids: $source.authorIds) object identified by id, batch size 2
+        author: [User] @hydrated(
+          service: "UserService"
+          field: "userById"
+          arguments: [
+            {name: "id" value: "$source.authorIds"}
```

Comment:
That's an interesting scenario - the source is a list of values, and the hydration field takes a single value. I'm guessing we would strongly be against consumers using this sort of config in a real-life case?
Also, what would happen if `$source.authorIds` returned more than 1 item?

Reply:
Yea we would be strongly against it. But that's another story. We just ported this functionality over from Nadel. If we want, we can look into deprecating it in the future.